source/dlangbot/app.d

```diff
@@ -457,13 +458,31 @@ void agentShutdownCheck(HTTPServerRequest req, HTTPServerResponse res)
 
 //==============================================================================
 
-void contributorStats(HTTPServerRequest req, HTTPServerResponse res)
+void contributorStatsOverall(HTTPServerRequest req, HTTPServerResponse res)
+{
+    contributorStats(req, res);
+}
+
+void contributorStatsPerCycle(HTTPServerRequest req, HTTPServerResponse res)
+{
+    contributorStats(req, res, "2022-01-01 00:00:00.000000", "2022-03-01 00:00:00.000000");
```

Comment:
What do you think about using some date arithmetics to automatically get the current cycle?

Reply:
I don't know for sure how these cycles will change so I prefer to hard code them for now.